BaseClasses.py

```diff
@@ -327,7 +328,7 @@ def find_common_pool(players: Set[int], shared_pool: Set[str]) -> Tuple[
                             del (counters[player][item])
                 return counters, classifications
 
-            common_item_count, classifications = find_common_pool(group["players"], group["item_pool"])
+            common_item_count, self.group_classifications = find_common_pool(group["players"], group["item_pool"])
```

Comment:
This should *update* self.group_classifications, not assign it, right?

Reply:
i never actually initialize it before this point, but you're right since this function is called per group. i'm still not sure if this is the correct approach, and even if it is we should probably uuse Progression_skip_balancing? and only set that for non progression items i think?

Reply:
Your approach is the approach I like 2nd most and I can't merge my own :D